pyscitt/pyscitt/client.py

```diff
@@ -35,6 +35,13 @@ class SigningType(Enum):
     HTTP = "HTTP"
 
 
+class ReceiptType(Enum):
+    """Receipt types supported by the ledger."""
+
+    EMBEDDED = "EMBEDDED"
+    RAW = "RAW"
```

Comment:
Assuming you can use the enums directly above, you can convert these to lowercase strings:

```suggestion
    EMBEDDED = "embedded"
    RAW = "raw"
```

Reply:
done